charts/internal/shoot-system-components/charts/csi-driver-node/templates/clusterrole-csi-volume-modifier.yaml

```diff
@@ -13,3 +13,6 @@ rules:
   - apiGroups: [""]
     resources: ["events"]
     verbs: ["create", "patch", "update"]
+  - apiGroups: [ "coordination.k8s.io" ]
+    resources: [ "leases" ]
+    verbs: [ "get", "list", "watch", "update", "patch" ]
```

Comment:
This shouldn't be needed because of https://github.com/gardener/gardener-extension-provider-aws/blob/master/charts/internal/shoot-system-components/charts/csi-driver-node/templates/role-csi-volume-modifier.yaml